README.rst

```diff
@@ -23,34 +16,47 @@ Emacs Lisp as well as Python.
    :target: https://stable.melpa.org/#/elpy
 
 
-Documentation
-=============
+==========================
+Elpy, the Emacs Python IDE
+==========================
 
-Elpy is fully documented at readthedocs.io:
+Elpy is an Emacs package to bring powerful Python editing to Emacs.
+It combines and configures a number of other packages, both written in
+Emacs Lisp as well as Python. Elpy is fully documented at `Readthedocs`_.
 
-https://elpy.readthedocs.io/en/latest/index.html
+.. _Readthedocs: https://elpy.readthedocs.io/en/latest/index.html
 
-Quick Installation
-==================
+Features
+========
 
-Python packages
----------------
+- `Code completion`_
+- `Code Navigation`_
+- `Interactive Python shell`_
+- `Virtualenv support`_
+- `On-the-fly syntax checking`_
+- `Access to documentation`_
+- `Debugging`_
+- `Test running`_
+- `Profiling`_
+- Snippet Expansion (using `yasnippet`_)
+- Code hinting
 
-First, install the required Python packages:::
+.. _Code completion: https://elpy.readthedocs.io/en/latest/ide.html#completion
+.. _Code Navigation: https://elpy.readthedocs.io/en/latest/ide.html#navigation
+.. _On-the-fly syntax checking: https://elpy.readthedocs.io/en/latest/ide.html#syntax-checking
+.. _Interactive Python shell: https://elpy.readthedocs.io/en/latest/ide.html#interactive-python
+.. _Access to documentation: https://elpy.readthedocs.io/en/latest/ide.html#documentation
+.. _Debugging: https://elpy.readthedocs.io/en/latest/ide.html#debugging
+.. _Test running: https://elpy.readthedocs.io/en/latest/ide.html#testing
+.. _Profiling: https://elpy.readthedocs.io/en/latest/ide.html#profiling
+.. _Virtualenv support: https://elpy.readthedocs.io/en/latest/concepts.html#virtual-envs
+.. _yasnippet: https://github.com/joaotavora/yasnippet
 
-  # Completion and code navigation
-  pip install jedi
-  # Code checks
-  pip install flake8
-  # Automatic formatting (PEP8, Yapf or Black)
-  pip install autopep8
-  pip install yapf
-  pip install black
 
-One-line install: ``pip install jedi flake8 autopep8 black yapf``
+Installation
+============
 
-With `use-package`
-------------------
+The most straightforward way of installing elpy is to use `use-package`:
```

Comment:
@galaunay for now ;-)  If the Debian package added a method to hook into python-mode (as-needed), `apt` would install an Elpy that was fully enabled and integrated out-of-the-box.  I'm also considering changing the Debian installation instructions to use the older `apt-get`, because then the `--install-suggests` can be added to the command, which will install all the (optional) Python deps.  IIRC our venvwrapper also defaults to `togglesitepackages`, but I'll need to double check and get back to you on  that point :-)

Reply:
That would be nice indeed to have just to install one package to get Elpy to work !
I would be happy to update the doc when it's working.